app/libs/Utils/HTMLHelpers.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+if (!function_exists('style_to')) {
+    function style_to($path, $attributes = [])
+    {
+        $url = asset($path);
+        $attributesString = '';
+
+        foreach ($attributes as $key => $value) {
```

Comment:
please add as default attr 
this 
https://github.com/LaravelCollective/html/blob/cb9092527e8b6cd8f9a69aedff70a3a3d95228e5/src/HtmlBuilder.php#L95

Reply:
done